x/evm/types/chain_config.go

```diff
@@ -4,46 +4,13 @@ import (
 	"math/big"
 	"strings"
 
-	"gopkg.in/yaml.v2"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/params"
 )
 
-// ChainConfig defines the Ethereum ChainConfig parameters using sdk.Int values instead of big.Int.
-//
-// NOTE 1: Since empty/uninitialized Ints (i.e with a nil big.Int value) are parsed to zero, we need to manually
-// specify that negative Int values will be considered as nil. See getBlockValue for reference.
-//
-// NOTE 2: This type is not a configurable Param since the SDK does not allow for validation against
-// a previous stored parameter values or the current block height (retrieved from context). If you
-// want to update the config values, use an software upgrade procedure.
-type ChainConfig struct {
```

Comment:
chainconfig is now defined in the protobuf go file? how come some types are defined twice, once in the ethermint library and again in the pb files while others (like chainconfig) only gets defined once?